src/Robot/RobotsFile.php

```diff
@@ -44,6 +44,10 @@ public function __construct($content)
     public function current()
     {
         $cur = $this->lineIterator->current();
+
+        if (!is_array($cur) || !array_key_exists(1, $cur)) {
```

Comment:
Could this be simplified to 

```php
    public function current()
    {
        $cur = $this->lineIterator->current();
        return is_array($cur) && count($cur) > 1 ? $cur[1] : '';
    }
```

or are there scenarios in which `count()` will be >1 but there won't be a key `1`? (it looks like we're populating the array with only numeric keys in the constructor)

Reply:
The problem is that the input :
 
```     *::before, <-- THIS ```

gets converted to : 

```
  array(2) {
    [0] =>
    string(1) "*"
    [2] =>
    string(16) "before, <-- THIS"
```

So there's nothing in $cur[1] ... hence the notice.